modules/jetstream-maxtext-deployment/variables.tf

```diff
@@ -31,8 +31,8 @@ variable "maxengine_deployment_settings" {
 
     model_name              = string           // Name of your LLM (for example: "gemma-7b")
     parameters_path         = string           // Path to the paramters for your model
-    metrics_port            = optional(number) // Emit Jetstream metrics on this port of each container
-    metrics_scrape_interval = optional(number) // Interval for scraping metrics (default: 10s)
+    metrics_port            = optional(number) // Emit Jetstream server metrics on this port of each container, no server metrics will be emitted if not set
```

Comment:
will you add these additional details to the README description as well?

Reply:
Added